src/config.py

```diff
@@ -228,6 +321,19 @@ def validate(cls, attribute_map: Mapping[str, Value]) -> List[str]:
                 'received only one dimension attribute. Please supply both "height_px" and "width_px", or neither.'
             )
 
+        # Validate manual focus
+        validate_attr_type("manual_focus", "number_value", attribute_map)
+        manual_focus = attribute_map.get(key="manual_focus", default=None)
+        if manual_focus:
+            if "color" not in sensor_list:
```

Comment:
[q] Does this depend on the camera module used? Some color cameras may be fixed focus?

Reply:
I think since we're limiting this attribute to the oak-d model, which is autofocus, we should be fine
<img width="594" alt="Screenshot 2024-10-23 at 10 21 02 AM" src="https://github.com/user-attachments/assets/2a5ce00e-64a5-4171-b326-4227b145fc35">